agent/constants/constants.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package constants
+
+const (
+	// AgentConfigFilename is the default file name of used for the agent
+	// config.
+	AgentConfigFilename = "agent.conf"
```

Comment:
Is this to avoid a circular dependency?

Reply:
Yeah it is. There is no way around this.